### ISSUE TITLE
Option to decently order microcode entries?

### ISSUE THREAD

**Issue by mirh** [platomav/MCExtractor#45]
At the moment AFAICT they are shown in whatever order they are placed inside a file. 
But at least for the *mcupdates* windows ships with, there's absolutely no regularity in the position of any single "entry" across different "whole file" releases. 
This makes diffing slightly more annoying than I'd like. 

Now, I don't really know how much important offset ordering is (so maybe changing this could not be a good default), but I'd like to see some way to have CPUID ordering.

**Comment by platomav** [comment]
Yes, results are shown in the order they were regex matched, meaning offset based. It is what makes sense for an _extractor_ utility as well. This can be solved in two different ways: adding support for a parsable output (e.g. json) or comparing the extracted firmware binaries themselves. Adding json is the proper way, but it requires time, so it's not  priority for me right now. I'll add it to the "todo" list though. For now, you can compare the output folders directly.
